### DISCUSSION TITLE
Is it possible to scan GTIN or SSCC codes?

### DISCUSSION THREAD

**Discussion by dmytroborbotko** [capawesome-team/capacitor-mlkit#104] [Q&A] [answered]
I set up the the test app and it works greate, but when i try to scan SSCC barcode, nothing happens. Could you please help me is it possible to implement this type for scanning?

**Comment by robingenz** [comment, marked as answer]
We only support this barcode formats: https://capawesome.io/plugins/mlkit/barcode-scanning/#barcodeformat

If you want to request support for another barcode format, please create a feature request in the [ML Kit repo](https://github.com/googlesamples/mlkit).
